src/averageAges.js

```diff
@@ -20,6 +20,15 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const filteredMen = people.filter(
+    person => person.sex === 'm'
+    && (!century || century === Math.ceil(person.died / 100)));
+
+  const filteredMenAge = filteredMen.map(year => year.died - year.born);
```

Comment:
Мапиш людей, а не роки
```suggestion
  const filteredMenAge = filteredMen.map(person => person.died - person.born);
```
```suggestion
  const filteredMenAge = filteredMen.map(year => year.died - year.born);
```